docs/source/cli.rst

```diff
@@ -49,7 +49,41 @@ specified.
 Experiments
 -----------
 
-Subcommands to create and list experiments.
+Subcommands to manage experiments.
+
+
+Create
+------
+Subcommand to create a new experiment using name. System will generate a unique ID for each
+experiment. Additionally, users can provide an artifact location  using ``-l`` or
+``--artifact-location`` option. If not provided, backend store will pick default location.
+
+All artifacts related to this experiment will be stored under artifact location under specific
+run directories.
+
+
+List
+----
+
+Listing of all experiments managed by backend store.
+
+
+Delete
+------
+
+Delete an active experiment. Command takes an mandatory argument experiment ID. If experiment
+is already deleted or not found, the command will throw error. This deletes associated metadata,
+runs and data as well. If the backend store controls locations of artifacts, they will be deleted
+as well. Deleted experiments can be restored using ``restore`` command.
```

Comment:
Maybe talk about how they can clear the trash here? Can also recommend setting up a cron job or something.